common/components/WalletDecrypt/components/InsecureWalletWarning.tsx

```diff
@@ -61,38 +62,31 @@ export class InsecureWalletWarning extends React.Component<Props, State> {
 
     return (
       <div className="WalletWarning">
-        <h2 className="WalletWarning-title">
-          This is <u>not</u> a recommended way to access your wallet
-        </h2>
+        <h2 className="WalletWarning-title">{translateRaw('INSECURE_WALLET_TYPE_TITLE')}</h2>
         <p className="WalletWarning-desc">
-          Entering your {walletType} on a website is <strong>dangerous</strong>. If our website is
-          compromised, or you accidentally visit a phishing website, you could{' '}
-          <strong>lose all of your funds</strong>. Before you continue, please consider:
+          {translateRaw('INSECURE_WALLET_TYPE_DESC', { var_wallet_type: walletType as string })}
         </p>
         <ul className="WalletWarning-bullets">
           <li>
-            Using <HelpLink article={HELP_ARTICLE.MIGRATE_TO_METAMASK}>MetaMask</HelpLink> or a{' '}
-            <HelpLink article={HELP_ARTICLE.HARDWARE_WALLET_RECOMMENDATIONS}>
-              Hardware Wallet
-            </HelpLink>{' '}
-            to access your wallet
+            {translateMarkdown('INSECURE_WALLET_RECOMMEND_1', {
+              var_metamask_article: knowledgeBaseURL + HELP_ARTICLE.MIGRATE_TO_METAMASK,
+              var_hardware_wallet_article:
+                knowledgeBaseURL + HELP_ARTICLE.HARDWARE_WALLET_RECOMMENDATIONS
+            })}
           </li>
           <li>
-            <HelpLink article={HELP_ARTICLE.RUNNING_LOCALLY}>
-              Downloading MyCrypto and running it offline & locally
-            </HelpLink>
+            {translateMarkdown('INSECURE_WALLET_RECOMMEND_2', {
+              var_run_local_article: knowledgeBaseURL + HELP_ARTICLE.RUNNING_LOCALLY
+            })}
           </li>
           <li>
-            Reading{' '}
-            <HelpLink article={HELP_ARTICLE.SECURING_YOUR_ETH}>
-              How to Protect Yourself and Your Funds
-            </HelpLink>
+            {translateMarkdown('INSECURE_WALLET_RECOMMEND_3', {
+              var_secure_your_eth_article: knowledgeBaseURL + HELP_ARTICLE.SECURING_YOUR_ETH
```

Comment:
I've noticed most of these translation variables are prefixed with `var_`, is there any reason for that?

Reply:
just to denote that that represents a variable. Using `var_` as a prefix is just a preference.